README.md

```diff
@@ -2,21 +2,29 @@
 [![codecov](https://codecov.io/gh/owncloud/owncloud-sdk/branch/master/graph/badge.svg)](https://codecov.io/gh/owncloud/owncloud-sdk)
 [![docs](https://img.shields.io/badge/api_docs-online-blue.svg)](https://owncloud.github.io/owncloud-sdk/)
 
-<p align="center">
-  <img src="https://i.imgur.com/9mKra3O.png" />
-</p>
+# ownCloud JavaScript SDK
 
-Use this light-weight JS library with a promise-based interface for seamless communication with your ownCloud instance.<br>
-Supports only browser JS but requires JavaScript techniques like webpack to compile it properly.<br>
-For Node.js please have a look at [this tag](https://github.com/owncloud/owncloud-sdk/releases/tag/last-nodejs).
+Use this light-weight JS library with a promise-based interface for seamless communication with your ownCloud instance.
+Supports only browser JS but requires JavaScript techniques like webpack to compile it properly.
```

Comment:
Afaik it also supports node since we switched from sabredav to webdav client lib.

Reply:
Changed the wording to include Node as supported use case, CI perhaps had some hickup? 🤷🏽 it's green now